CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [5.1.2] - TBD
```

Comment:
I think this is a fairly big change so we should go with at least 5.2.0 as the version number. Can you check what happens if you take a table replicated using the previous version which has the column duplicated and then replicate it using this version - does the replication work? is the data queryable? does it return data partitioned as expected and with the values filled in properly etc?

Reply:
It says the replication fails `Caused by: com.hotels.housekeeping.HousekeepingException: Unable to schedule path [path] for deletion`. But I checked the table and the schema has been updated and I ran a query on it and it worked...?!

Reply:
Looks like the replication works, but housekeeping has failed. Did you set the orphaned data strategy in the settings? or configure a housekeeping db?

Reply:
No, havent done anything other than basic replication config

Reply:
Turned off housekeeping and replicates fine now 👍